README.md

```diff
@@ -99,6 +99,12 @@ export const useStore = defineStore('store', {
 
 All the available configuration options are explained [here](https://prazdevs.github.io/pinia-plugin-persistedstate/guide/config.html).
 
+## Nuxt3
+
+Are you using [Pinia](https://pinia.vuejs.org/) with [Nuxt](https://nuxt.com/) ([Pinia in Nuxt](https://pinia.vuejs.org/ssr/nuxt.html)) and want to add the persisted state plugin? There is a [plugin for that](https://prazdevs.github.io/pinia-plugin-persistedstate/frameworks/nuxt-3.html).
+
+> **Note:** There is currently a [bug](https://github.com/prazdevs/pinia-plugin-persistedstate/issues/236) with a [workaround](https://github.com/prazdevs/pinia-plugin-persistedstate/issues/236#issuecomment-1711423612) affecting Nuxt 3.7+.
+

```

Comment:
No need for so many links (might pollute for screen readers).
Also maybe it should sound less "ad-like" and be more straightforward :)
And it's a Nuxt module (not a plugin). 

Also not sure about adding info about a bug in 3.7 that is not sure to be on our part 🤔